db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Item.create!(title: 'Power Rangers', description: 'Go Go Power Rangers! WEOSWWWEAHHWOOAWOAHWOAH')
+Item.create!(title: 'Magic School Bus', description: 'Ms. Frizzle taught us science')
+Item.create!(title: 'Fall of the Soviet Union', description: 'A victory for capitalism and the free world')
```

Comment:
Nice seedfile. Although, why use `create!` instead of regular old `.create`? Just because it will fail noisy, if it fails?